<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SlideBox1</class>
 <widget class="QWidget" name="SlideBox1">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>350</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>350</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">SlideBox1{background-color:transparent}</string>
  </property>
  <widget class="QPushButton" name="fakeButton1">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>20</y>
     <width>300</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image:url(:/assets/image/titlebox1.png);
font: 75 12pt &quot;ADMUI3Lg&quot;;
background-color:transparent;
color:white</string>
   </property>
   <property name="text">
    <string>PushButton1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="fakeButton2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>90</y>
     <width>300</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image:url(:/assets/image/titlebox1.png);
background-color:transparent;
color:white</string>
   </property>
   <property name="text">
    <string>PushButton2</string>
   </property>
  </widget>
  <widget class="QPushButton" name="fakeButton3">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>160</y>
     <width>300</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image:url(:/assets/image/titlebox1.png);
background-color:transparent;
color:white</string>
   </property>
   <property name="text">
    <string>PushButton3</string>
   </property>
  </widget>
  <widget class="QPushButton" name="fakeButton4">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>240</y>
     <width>350</width>
     <height>80</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image:url(:/assets/image/titlebox1.png);
background-color:transparent;
color:white</string>
   </property>
   <property name="text">
    <string>PushButton4</string>
   </property>
  </widget>
  <widget class="QPushButton" name="fakeButton5">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>350</y>
     <width>300</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image:url(:/assets/image/titlebox1.png);
background-color:transparent;
color:white</string>
   </property>
   <property name="text">
    <string>PushButton5</string>
   </property>
  </widget>
  <widget class="QPushButton" name="fakeButton6">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>420</y>
     <width>300</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image:url(:/assets/image/titlebox1.png);
background-color:transparent;
color:white</string>
   </property>
   <property name="text">
    <string>PushButton6</string>
   </property>
  </widget>
  <widget class="QPushButton" name="fakeButton7">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>490</y>
     <width>300</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image:url(:/assets/image/titlebox1.png);
background-color:transparent;
color:white</string>
   </property>
   <property name="text">
    <string>PushButton7</string>
   </property>
  </widget>
  <widget class="QPushButton" name="upPage">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>5</y>
     <width>320</width>
     <height>220</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="downPage">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>335</y>
     <width>320</width>
     <height>220</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="chooseButton">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>240</y>
     <width>350</width>
     <height>80</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
